solutions/cf-1702E.mdx

```diff
@@ -0,0 +1,128 @@
+---
+id: cf-1702E
+source: CF
+title: Split Into Two Sets
+author: Aadit Ambadkar
+---
+
+[Official Editorial (C++)](https://codeforces.com/blog/entry/104763)
+
+## Explanation
+
+**Time Complexity:** $\mathcal{O}(n)$
+
+First observe that if any number is present on more than 2 dominoes or if a dominoe has the same number repeated twice, then 
+it is impossible to create the 2 sets.
```

Comment:
why?

Reply:
that's kind of obvious if you read the problem and the sample input

Reply:
"left as an exercise to the reader" moment

Reply:
honestly imma put that there